_episodes/51.md

```diff
@@ -0,0 +1,101 @@
+
+## Releases 423-427
+
+To be filled out shortly...
+
+## Concept of the episode: PopSQL
+
+It may be familiar to some of our viewers to describe an environment where
+key queries and dashboards are buried in someone's personal workspace, and you
+have to go ask them directly every time you want to check on a tool's health.
+When you're running a world-class, highly-performant query engine like Trino and
+investing time and resources into maintaining it, shouldn't you treat the end
+product like a first-class, collaborative, versioned system, too?
+
+[PopSQL](https://popsql.com/), a playful spin on the word popsicle, solves the
+sadness that is disorganized and siloed insights by centralizing queries into a
+collaborative platform that has versioning, security, and a suite of
+collaborative tools comparable to Google Drive. Want to work with your teammate
+on a query? You can open up the same editor and see the same thing. Want to see
+what that query someone ran last week was to see how the new feature is doing?
+It's there. Have a suggestion to improve something? Leave a comment. Realize
+your suggestion was wrong and need to undo the change? You can view past
+versions of the query.
+
+PopSQL and Trino make sense together. PopSQL provides a best-in-class interface
+for organizing, collaborating, and working together on all of your SQL queries
+across the business, and Trino handles running those queries at unparalleled
+speeds. They go hand-in-hand for treating your data and SQL analytics as first
+class citizens. In today's episode, we'll be exploring what PopSQL is, how it
+integrates with Trino, and how the engineers at PopSQL have done some cool
+things with Trino to make the integration better than ever before. We'll start
+with that last one, actually.
+
+## Concept of the episode: A new Node.js adapter for Trino
+
+Trino in the frontend is... a tricky thing. We can go ahead and admit that the
+[Trino web UI](/docs/current/admin/web-interface.html) isn't going to win any
+awards for design or functionality. And while a couple Node-based clients exist
+out there, including [presto-client-node](https://www.npmjs.com/package/presto-client)
+and [lento](https://github.com/vweevers/lento), neither are great at streaming
+data, and they lack some interactivity that could be used to make an interface
+more robust. So when PopSQl's engineers went to build their frontend and
+integrate with Trino, what did they do? Build their own adapter.
+
+We'll talk about how it was implemented, what key features it unlocks, and why
+it makes using PopSQL with Trino an even better experience.
+
+## Demo of the episode: Using PopSQL with Trino
+
+It's hard to write show notes for a demo, because you can't really experience
+the demo by reading about what's happening. But as a surface-level overview,
+we'll be going over:
+
+* Setting up a connection
+* The schema explorer
+* The SQL editor
+* Query scheduling
```

Comment:
@colebow time permitting I'll show charts/dashboards too since they're more visually appealing and are typically the first thing people ask about as soon as they have a table of results back from the database